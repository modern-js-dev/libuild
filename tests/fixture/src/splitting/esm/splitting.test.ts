import { getLibuilderTest } from '@modern-js/libuild-test-toolkit';

describe('fixture:splitting', () => {
  it('esm', async () => {
    const bundler = await getLibuilderTest({
      root: __dirname,
      format: 'esm',
      splitting: true,
    });
    await bundler.build();
    bundler.expectJSOutputMatchSnapshot();
  });
});
