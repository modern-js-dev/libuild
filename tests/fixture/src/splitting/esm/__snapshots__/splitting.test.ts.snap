// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixture:splitting esm 1`] = `
"// index.ts
import(\\"./dynamic.js\\").then((value) => {
  console.log(value);
});
"
`;

exports[`fixture:splitting esm 2`] = `
"// dynamic.ts
function foo() {
  console.log(1);
}
export {
  foo
};
"
`;
